Documentation/using-rkt-with-systemd.md

```diff
@@ -58,7 +62,149 @@ MemoryLimit=1G
 Environment=HTTP_PROXY=192.0.2.3:5000
 Environment=STORAGE_PATH=/opt/myapp
 Environment=TMPDIR=/var/tmp
+# Fetch the app (not strictly required, `rkt run` will fetch the image if there is not one)
+ExecStartPre=/usr/bin/rkt fetch myapp.com/myapp-1.3.4
 # Start the app
 ExecStart=/usr/bin/rkt run --inherit-env --private-net --port=http:8888 myapp.com/myapp-1.3.4
 Restart=always
 ```
+
+## Using (not only) `systemd` tools
```

Comment:
The same, I prefer unquoted systemd. The same for all the commands below, that's what we do in other files like commands.md.